File: Linear/Static/Array/02_Inserting/Given_Position/Built-In/Inserting.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Scanner;

class InsertingGivenPositionBuiltIn {
    public static void main(String[] args) {
        ArrayList<Integer> arr = new ArrayList<>(Arrays.asList(10, 20, 30, 40));
        int x = 50;
        System.out.println("Array before insertion");
        for (int i = 0; i < arr.size(); i++) {
            System.out.println(arr.get(i) + " ");
        }

        Scanner sc = new Scanner(System.in);
        System.out.println("\nEnter a position: ");
        int pos = sc.nextInt();

        // Insert x at the given position.
        arr.add(pos - 1, x);

        sc.close();

        System.out.println("\nArray after insertion");
        for (int i = 0; i < arr.size(); i++) {
            System.out.println(arr.get(i) + " ");
        }
    }
}

File: Linear/Static/Array/02_Inserting/Given_Position/Built-In/Inserting.py
arr = [10, 20, 30, 40]
x = 50
print("Array before insertion")
for i in range(len(arr)):
    print(arr[i], end=" ")

pos = int(input("\nEnter a position: "))

# Insert x at the given position.
arr.insert(pos - 1, x)

print("\nArray after insertion")
for i in range(len(arr)):
    print(arr[i], end=" ")